packages/docs/src/components/DocSiteNavigation.tsx

```diff
@@ -136,22 +165,51 @@ const DocSiteSidebar = ({ isMobileNavOpen, location }: DocSiteNavProps) => {
   const navItems: VerticalNavItemProps[] = formatNavData(data?.allFile?.group);
 
   return (
-    <nav
-      className={classnames('ds-l-md-col--3 ds-u-padding--2 ds-u-fill--white c-sidebar', {
-        'c-sidebar--open': isMobileNavOpen,
+    <div
+      className={classnames('ds-l-md-col--3 ds-u-padding--0 ds-u-md-padding--2 c-navigation', {
+        'c-navigation--open': isMobile && isMobileNavOpen,
       })}
     >
-      <VerticalNav
-        className="c-nav"
-        items={navItems}
-        component={GatsbyLink}
-        selectedId={location ? location.pathname : ''}
-      />
-      <div className="ds-u-md-display--none ds-u-margin-top--2 c-sidebar__mobile-button-wrapper">
-        <GithubLinks />
+      <header className="c-navigation__header">
+        <Button
+          className="ds-u-md-display--none ds-u-padding-left--0 ds-u-padding-right--1"
+          variation="transparent"
+          aria-expanded={isMobileNavOpen}
+          aria-controls="c-mobile-navigation"
+          onClick={toggleMenu}
+        >
+          {isMobileNavOpen ? (
+            <CloseIconThin className="ds-u-font-size--xl" />
+          ) : (
+            <MenuIconThin className="ds-u-font-size--xl" />
+          )}
+        </Button>
+        <h1 className="c-navigation__title">
+          <a href="/">CMS Design System</a>
+        </h1>
+      </header>
+
+      <div
+        id="c-mobile-navigation"
+        // hidden attr applied on mobile breakpoints when nav is closed
+        hidden={isMobile && !isMobileNavOpen}
+        className="ds-u-padding--2 ds-u-md-padding--0"
+      >
+        <VerticalNav
+          className="c-navigation__link-list"
+          items={navItems}
+          component={GatsbyLink}
+          selectedId={location ? location.pathname : ''}
+        />
+        <a
+          href="https://github.com/CMSgov/design-system"
+          className="c-navigation__github-link ds-c-link"
+        >
+          View code on GitHub
```

Comment:
Are we going to include the GitHub logo here?

Reply:
We can for sure! Let me know if you need me to grab the logo from font awesome